mobile-app/app/components/BottomSheetTokenList.tsx

```diff
@@ -142,89 +138,106 @@ export const BottomSheetTokenList = ({
                     thousandSeparator
                     displayType="text"
                     renderText={(value) => (
-                      <ThemedText
-                        light={tailwind("text-gray-700")}
-                        dark={tailwind("text-gray-300")}
+                      <ThemedTextV2
+                        style={tailwind("text-sm font-semibold-v2 text-right")}
+                        light={tailwind("text-mono-light-v2-900")}
+                        dark={tailwind("text-mono-dark-v2-900")}
                         testID={`select_${item.token.displaySymbol}_value`}
                       >
                         {value}
-                      </ThemedText>
+                      </ThemedTextV2>
                     )}
                   />
-                  <ActiveUSDValue
+                  <ActiveUSDValueV2
                     price={new BigNumber(item.available).multipliedBy(
                       activePrice
                     )}
-                    containerStyle={tailwind("justify-end")}
+                    containerStyle={tailwind("justify-end mt-1")}
                     isOraclePrice={isOraclePrice}
                   />
                 </View>
-                <ThemedIcon
-                  iconType="MaterialIcons"
-                  name="chevron-right"
-                  size={20}
-                />
               </View>
-            </ThemedTouchableOpacity>
+            </ThemedTouchableOpacityV2>
           );
         }}
-        ListHeaderComponent={
-          <ThemedView
-            light={tailwind("bg-white border-gray-200")}
-            dark={tailwind("bg-gray-800 border-gray-700")}
-            style={tailwind(
-              "flex flex-row justify-between items-center px-4 py-2 border-b",
-              { "py-3.5 border-t -mb-px": Platform.OS === "android" }
-            )} // border top on android to handle 1px of horizontal transparent line when scroll past header
-          >
-            <ThemedText style={tailwind("text-lg font-medium")}>
-              {headerLabel}
-            </ThemedText>
-            <TouchableOpacity onPress={onCloseButtonPress}>
-              <ThemedIcon iconType="MaterialIcons" name="close" size={20} />
-            </TouchableOpacity>
-          </ThemedView>
-        }
-        stickyHeaderIndices={[0]}
         keyExtractor={(item) => item.tokenId}
         style={tailwind({
-          "bg-gray-800": !isLight,
-          "bg-white": isLight,
+          "bg-mono-dark-v2-100": !isLight,
+          "bg-mono-light-v2-100": isLight,
+          "pt-1 -mt-1": Platform.OS === "android", // Word-around fix for line showing on android
         })}
+        contentContainerStyle={tailwind("px-5 pb-20")}
       />
     );
   });
+
+export function CollateralFactorTag({
+  factor,
+}: {
+  factor?: string;
+}): JSX.Element | null {
+  const DEFAULT_FACTOR = "1";
+  return factor !== undefined && factor !== DEFAULT_FACTOR ? (
```

Comment:
can we use BigNumber's `isEqualTo` just in case api suddenly returns `1.00` in the future